src/Microsoft.ML.AutoML/Experiment/Runners/CrossValSummaryRunner.cs

```diff
@@ -8,6 +8,7 @@
 using System.Linq;
 using Microsoft.ML.Data;
 using Microsoft.ML.Runtime;
+using Microsoft.ML.Trainers.FastTree;
```

Comment:
Should be removed, otherwise a user needs the FastTree NuGet (and the ability to run the unmanaged code). Likely was needed before the [`CumulativeSum`](https://github.com/dotnet/machinelearning/blob/712c3ec0745f45b93e394f8e333deaa5da4f2737/src/Microsoft.ML.FastTree/Utils/LinqExtensions.cs#L202-L215) function was copied directly in to this code.